client_documents_test.go

```diff
@@ -9,6 +9,12 @@ type docTest struct {
 	Name string `json:"name"`
 }
 
+type docTestBooks struct {
+	BookId int    `json:"book_id"`
```

Comment:
Ahhhh So close to the goal :laughing:  
BookdID not BookId (in go an acronym is always in uppercase)